semgrep/java/java-use-of-des-encryption-algorithm.yml

```diff
@@ -0,0 +1,25 @@
+- id: java-use-of-des-encryption-algorithm
+  message: |
+    DES is a weak algorithm that has known attacks and can be broken. Consider using a secure algorithm such as AES.
```

Comment:
Is it possible to add an auto-fix for this rule? Not sure how the fix should look like so it is hard to say..